lib/iris/cube.py

```diff
@@ -3368,8 +3368,7 @@ def aggregated_by(self, coords, aggregator, **kwargs):
 
         A "group coordinate" is a coordinate where repeating values represent a
         single group, such as a month coordinate on a daily time slice.
-        TODO: It is not clear if repeating values must be consecutive to form a
-        group.
+        Repeated values can form a group even if they are not consecutive.
```

Comment:
Hi @rhattersley ! Many thanks for that, that's an excellent point. I'll make the appropriate change and put through an update very shortly...